src/psyclone/gocean1p0.py

```diff
@@ -396,67 +398,68 @@ def __init__(self, parent=None,
 
         # Loop bounds for a mesh with NE offset
         self._bounds_lookup['offset_ne']['ct']['all_pts'] = \
-            {'inner': {'start': "1", 'stop': "+1"},
-             'outer': {'start': "1", 'stop': "+1"}}
+            {'inner': {'start': "1", 'stop': "{stop}+1"},
```

Comment:
It feels better/more consistent to have `{stop}+1` rather than just `+1` - thanks.